backend/src/question-service/controllers/questions.ts

```diff
@@ -12,6 +12,28 @@ export const getQuestions: RequestHandler = async (req, res, next) => {
   }
 };
 
+export const getQuestion: RequestHandler[] = [
+  param("questionId").notEmpty().withMessage("question field cannot be empty."),
+  param("questionId").isNumeric().withMessage("questionId should be a number."),
```

Comment:
by the way, these are chainable e.g.

```js
param("questionId")
    .notEmpty()
    .withMessage("question field cannot be empty.")
    .isNumeric()
    .withMessage("questionId should be a number."),
```

and i think if questionId is empty `localhost:8081/api/questions/`
the index handler will run instead :P

Reply:
Oh! Cool thank you : >